server/src/main/java/org/apache/druid/catalog/model/ColumnSpec.java

```diff
@@ -49,14 +48,10 @@ public class ColumnSpec
   private final String name;
 
   /**
-   * The data type of the column expressed as a supported SQL type. The data type here must
-   * directly match a Druid storage type. So, {@code BIGINT} for {code long}, say.
-   * This usage does not support Druid's usual "fudging": one cannot use {@code INTEGER}
-   * to mean {@code long}. The type will likely encode complex and aggregation types
-   * in the future, though that is not yet supported. The set of valid mappings is
-   * defined in the {@link Columns} class.
+   * The data type of the column expressed as a supported Druid type. The data type here must
+   * directly match a Druid storage type.
    */
-  private final String sqlType;
+  private final String dataType;
```

Comment:
What is the reasoning behind changing from sql type to druid type here?

Reply:
It now accepts both SQL and Druid native type strings in the catalog definition